account_statement_import_online_qonto/README.rst

```diff
@@ -0,0 +1 @@
+Will be regenerated from the readme subdir
```

Comment:
Don't erase the README. This makes double diff and increase uselessly the size of the repo: one for removing content, and the other for adding it again.